rules/o365/credential_access_o365_potential_password_spraying_attack.toml

```diff
@@ -0,0 +1,53 @@
+[metadata]
+creation_date = "2020/12/01"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/12/01"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies a high number of failed O365 user authentication attempts from a single IP address, which could be indicative
+of a password spraying attack. An adversary may attempt a password spraying attack to obtain unauthorized access to user
+accounts.
+"""
+false_positives = [
+    """
+    Automated processes that attempt to authenticate using expired credentials and unbounded retries may lead to false
+    positives.
+    """,
+]
+from = "now-30m"
+index = ["filebeat-*"]
+language = "kuery"
+license = "Elastic License"
+name = "Potential Password Spraying of O365 User Accounts"
+note = "The O365 Fleet integration or Filebeat module must be enabled to use this rule."
+risk_score = 73
+rule_id = "3efee4f0-182a-40a8-a835-102c68a4175d"
+severity = "high"
+tags = ["Elastic", "Cloud", "Office 365", "Continuous Monitoring", "SecOps", "Identity and Access"]
```

Comment:
I noticed where Microsoft rebranded last April to `Microsoft 365`, do we want to align our tags and wording around that instead of Office 365?

Reply:
That's a good question. I see both in certain places. I'll run it by PM for wording.